models/klaviyo__person_campaign.sql

```diff
@@ -0,0 +1,22 @@
+with person_campaign as (
```

Comment:
Would it be beneficial to combine person campaign and flow together? I think that would paint a whole picture for the person opposed to having flow and campaign person data in two separate tables? I could see customers possibly wanting to combine this. What are your thoughts?